_includes/nav-bar-blog.html

```diff
@@ -1,32 +1,66 @@
 <div class="header">
```

Comment:
why is there a different nav bar for the blog? this does not seem necessary.

Reply:
it's old naming but it's deferend nav-bar from homepage nav-bar, I will fix the naming